types/maxmsp/maxmsp-tests.ts

```diff
@@ -47,8 +47,16 @@ folder.close();
 
 // maxmsp-tests.ts
 
-const glbl = new Global('myGlobal');
-(<any> glbl).prop = 'hello';
+class MyGlobal extends Global {
```

Comment:
I think this is a better way of approaching this and doen't require to change the Global class definition.

In the previous example anything under `glbl` is `any` leaving the door open whatever might happen at runtime

Reply:
Tdaoy I gave it a test to this internally and I keep going back and forth, I am tending to think the previous approach was better, having to subclass just to get around typescript is not really good either.